docs/vscode.md

```diff
@@ -3,50 +3,106 @@
+-   [Debugger for Firefox](https://marketplace.visualstudio.com/items?itemName=firefox-devtools.vscode-firefox-debug)
+-   [Debugger for Chrome](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome)
+-   [GitLens](https://marketplace.visualstudio.com/items?itemName=eamodio.gitlens)
+-   [yarn](https://marketplace.visualstudio.com/items?itemName=gamunu.vscode-yarn)
+-   [vscode-icons](https://marketplace.visualstudio.com/items?itemName=vscode-icons-team.vscode-icons)
+-   [export-index](https://marketplace.visualstudio.com/items?itemName=BrunoLM.export-index)
 
 The ESLint and Prettier integrations can be configured to fix and format your code on save.
 The Python integration can be configured to use Black as a formatter and run checks with Pylint,
 whether they are globally installed or in a virtualenv. Sample `.vscode/settings.json`:
 
 ```json
 {
-  "[typescript]": {
-    "editor.formatOnSave": true
-  },
-  "[typescriptreact]": {
-    "editor.formatOnSave": true
-  },
-  "[python]": {
-    "editor.formatOnSave": true
-  },
-  "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true,
-    "source.organizeImports": false
-  },
-  "python.testing.pytestArgs": [
-    "flask"
-  ],
-  "python.testing.unittestEnabled": false,
-  "python.testing.nosetestsEnabled": false,
-  "python.testing.pytestEnabled": true,
-  "python.formatting.provider": "black",
-  "python.formatting.blackPath": "flask/venv/bin/black",
-  "python.linting.pylintEnabled": true,
-  "python.linting.enabled": true
+    "[typescript]": {
+        "editor.formatOnSave": true
+    },
+    "[typescriptreact]": {
+        "editor.formatOnSave": true
+    },
+    "[python]": {
+        "editor.formatOnSave": true
+    },
+    "editor.codeActionsOnSave": {
+        "source.fixAll.eslint": true,
+        "source.organizeImports": false
+    },
+    "python.testing.pytestArgs": ["flask"],
+    "python.testing.unittestEnabled": false,
+    "python.testing.nosetestsEnabled": false,
+    "python.testing.pytestEnabled": true,
+    "python.formatting.provider": "black",
+    "python.formatting.blackPath": "flask/venv/bin/black",
+    "python.linting.pylintEnabled": true,
+    "python.linting.enabled": true
 }
 ```
+
+## VSCode debugger for flask container
+
+VSCode debugger: [https://code.visualstudio.com/docs/editor/debugging]
+VSCode & remote containers: [https://code.visualstudio.com/docs/remote/containers]
+Using profiels with Compose: [docs.docker.com/compose/profiles]
+
+To set up a development environment with Flask, Docker and VSCode when the docker context points to the local machine, see the steps below. If the Flask application container is running on the VM, an additional step is required to set the local docker context to point to the docker instance on the VM.
+
+To directly enter a Docker development that supports debugging in the container and launch the debugger:
+
+-   Step 1: `docker-compose up -d mysql minio app_debug`
```

Comment:
Should only need `docker-compose up -d app_debug` since the dependencies are specified in the compose file.